layout/_macro/sidebar.swig

```diff
@@ -29,10 +29,12 @@
       <section class="site-overview-wrap sidebar-panel{% if not display_toc or toc(page.content).length <= 1 %} sidebar-panel-active{% endif %}">
         <div class="site-overview">
           <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
-            {% if theme.avatar %}
+            {% if theme.avatar.url %}
+              <a href="/" class="site-author-image" rel="start" style="border:none">
```

Comment:
Ok, what about `a href`? Is it needed or mb not? And if yes, mb need to add additional option to redefine this url?

Reply:
I prefer to add the function `clicking the avatar to homepage` as the inherent function. Besides, i don't know how to add this in `if condition`. 😢

Reply:
You mean 

- the default to homepage?
- the custom to custom linking?

Reply:
No, i mean:

* the default for nothing (as it for now)
* custom - is custom; if user want, he can define link

Reply:
How about this one?

- the default for nothing, but linking to the present page.
- user can custom the external url of avatar.

Reply:
No need to use `exturl` option.

Reply:
well, I delete the linking function of avatar.